python_modules/dagster/dagster_tests/cli_tests/test_project_commands.py

```diff
@@ -90,6 +90,28 @@ def test_from_example_command_succeeds():
         assert not os.path.exists("my_dagster_project/tox.ini")
 
 
+def test_from_example_command_versioned_succeeds():
```

Comment:
Nice

A couple other useful regression tests might be:
- testing to make sure it fails correctly if the path already exists
- testing to make sure it doesn't fail if a path that starts with the same prefix already exists (foo succeeds even if foobar exists)